### PR TITLE
Api implementation

### PR DESCRIPTION
**Pull request by huseyinbiyik**
You can find the features added with this Pull Request below.

- A new game created using the API
- "Refresh" button was implemented (receiving data from the API and parsing the JSON).
-  "Submit" button on the from implemented (sending data to the API).
-  async and await JavaScript features used to consume the API.

### CODE REVIEW

File: src/modules/postData.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export default async function postData(name, score) {
```
Comment (zainsadaqat) [inline src/modules/postData.js]: Kindly replace the ES5 function with the ES6 arrow function syntax. 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @huseyinbiyik  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/postData.js (hunk 1 above)
Comment (huseyinbiyik) [reply src/modules/postData.js]: Thank you.

File: src/index.html [hunk 2]
```diff
@@ -11,21 +11,18 @@ <h1>Leaderboard</h1>
       <section class="display-container">
         <header class="display-heading">
           <h2>Recent Scores</h2>
-          <button type="submit">Refresh</button>
+          <button id="refresh-button" type="submit">Refresh</button>
         </header>
         <div class="display-board">
```
Comment (Ismailco) [inline src/index.html:16]: - [x] [Optional] When you input a long score number the UI breaks, please consider adding some kind of input validation to fix this

Screenshot_20220518_180843

**Review by Ismailco — APPROVED** [review]
### Hi @huseyinbiyik,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 2 above)
Comment (huseyinbiyik) [reply src/index.html:16]: Thank you for the review and the recommendation.

**Pull request merged**